accounting/src/delta/delta_data_collection/mod.rs

```diff
@@ -19,19 +19,19 @@ use serialization::{Decode, Encode};
 
 use crate::error::Error;
 
+use self::undo::*;
+
```

Comment:
Minor: it looks a bit weird to `use` something before it is introduced. I would personally put `use` after the corresponding `mod`

Reply:
will add with my next PR